exchange/exchange-ps/exchange/Set-RetentionPolicy.md

```diff
@@ -128,10 +128,12 @@ The IsDefault switch specifies that this retention policy is the default retenti
 
 When you use this switch, you'll get a warning that this retention policy will replace the current default retention policy (the policy where the IsDefault property value is currently True) as the default. By default, the default retention policy is named Default MRM Policy.
 
-**Note**: Changing which retention policy is the default might or might not affect new and existing mailboxes based on the value of the RetentionPolicy parameter on the Set-MailboxPlan cmdlet:
+**Note**: Changing which retention policy is the default might or might not affect new and existing mailboxes based on the value of the RetentionPolicy parameter on the all mailbox plans:
 
 - $null (blank): Changes to which retention policy is the default are continually reflected in new and existing mailboxes.
-- A retention policy is specified: Changes to which retention policy is the default are ignored. The retention policy specified in the mailbox plan is assigned to new mailboxes, and changing which policy is specified in the mailbox plan doesn't affect existing mailboxes. If you set the value back to blank, new mailboxes will use the default policy going forward. Existing mailboxes created during the time when a retention policy was specified in the mailbox plan will continue to use that policy, and are unaffected by changes to which retention policy is the default. Any existing mailboxes that you created during the last time when there was no retention policy specified in the mailbox plan will be continually affected by changes to which retention policy is the default.
+- A retention policy is specified: The value of the RetentionPolicy parameter for all mailbox plans should either be $null (blank) or match the Exchange retention policy that is configured as default for the tenant otherwise the experience may be inconsistent when creating new mailboxes, enabling disabled mailboxes, and changing licenses.  For more information see [Mailbox plans in Exchange Online](/exchange/recipients-in-exchange-online/manage-user-mailboxes/mailbox-plans/).
```

Comment:
```suggestion
- A retention policy is specified: The value of the RetentionPolicy parameter for all mailbox plans should either be $null (blank) or match the Exchange retention policy that is configured as default for the tenant otherwise the experience may be inconsistent when creating new mailboxes, enabling disabled mailboxes, and changing licenses. For more information see [Mailbox plans in Exchange Online](/exchange/recipients-in-exchange-online/manage-user-mailboxes/mailbox-plans/).
```